adapter.go

```diff
@@ -271,8 +274,8 @@ func (a *Adapter) LoadPolicy(model model.Model) error {
 	return nil
 }
 
-func (a *Adapter) savePolicyLine(ptype string, rule []string) *CasbinRule {
-	line := &CasbinRule{PType: ptype, tableName: a.tableName}
+func (a *Adapter) genPolicyLine(ptype string, rule []string) *CasbinRule {
+	line := CasbinRule{PType: ptype, tableName: a.tableName}
```

Comment:
The return value uses a pointer - `return &line`.

Reply:
It is just a code style. It will return a confirmed type when data returning. We can choose to return value or pointer. Most time I think we need to return value to reduce `escape`. And this func return a pointer is a compatible way.

Reply:
> It is just a code style. It will return a confirmed type when data returning. We can choose to return value or pointer. Most time I think we need to return value to reduce `escape`. And this func return a pointer is a compatible way.

ok, I just has some curious